ftgo-application

sabahatatta opened the pull request:
Hi David,

I hope you are doing good. My name is Sabahat Atta. My area of research is Microservices and specifically Bad Smells in Microservices. Your research work has been a great help. I was also creating a dataset of the microservices based application and found your repository. 

Below is the link of another Microservice toy project based upon the book Microservices Patterns. You might want to include it. 
https://github.com/microservices-patterns/ftgo-application

Below are some other suggested projects to add in the dataset if they fulfill your criteria.

1- https://github.com/GoogleCloudPlatform/microservices-demo
2- https://github.com/microservices-practical
3- https://github.com/sivaprasadreddy/spring-boot-microservices-series
4- https://github.com/instana/robot-shop
5- https://github.com/microservices-demo/microservices-demo
6- https://github.com/microservices-demo
7- https://github.com/apssouza22/java-microservice
8- https://github.com/mfornos/awesome-microservices
9- https://github.com/microsoft/PartsUnlimitedMRPmicro
10- https://github.com/jlroo/microservice
11- https://github.com/cer/microservices-examples
12- https://github.com/fChristenson/microservices-example
13- https://github.com/Vanilla-Java/Microservices
14- https://github.com/ewolff/microservice-kubernetes
15- https://github.com/elgris/microservice-app-example
16- https://github.com/vmudigal/microservices-sample 

In addition, I would like to identify that there is a bit of a repetition of projects in the existing list. I hope that these toy/demo projects listed above can be added into the existing list. 

Best Regards,
Sabahat Atta

sabahatatta committed:
ftgo-application

Hi David,

I hope you are doing good. My name is Sabahat Atta and I am an MSCS research student at COMSATS university Islamabad, Lahore Campus. My area of research is Microservices and specifically Bad Smells in Microservices. Your research work has been a great help. I was also creating a dataset of the microservices based application and found your repository. 

Below is the link of another Microservice toy project based upon the book Microservices Patterns. You might want to include it. 
https://github.com/microservices-patterns/ftgo-application

Below are some other suggested projects to add in the dataset if they fulfill your criteria.

1- https://github.com/GoogleCloudPlatform/microservices-demo
2- https://github.com/microservices-practical
3- https://github.com/sivaprasadreddy/spring-boot-microservices-series
4- https://github.com/instana/robot-shop
5- https://github.com/microservices-demo/microservices-demo
6- https://github.com/microservices-demo
7- https://github.com/apssouza22/java-microservice
8- https://github.com/mfornos/awesome-microservices
9- https://github.com/microsoft/PartsUnlimitedMRPmicro
10- https://github.com/jlroo/microservice
11- https://github.com/cer/microservices-examples
12- https://github.com/fChristenson/microservices-example
13- https://github.com/Vanilla-Java/Microservices
14- https://github.com/ewolff/microservice-kubernetes
15- https://github.com/elgris/microservice-app-example
16- https://github.com/vmudigal/microservices-sample 

In addition, I would like to identify that there is a bit of a repetition of projects in the existing list. I hope that these toy/demo projects listed above can be added into the existing list. I would also like to communicate with you on my research topic if you can grant me some time from your valuable time.  Thanking you in advance for your cooperation in this regard.

Best Regards,
Sabahat Atta

davidetaibi commented:
Thanks for your PR. 
I have added the ftgo application. 

there are several interesting projects in your list that could be added, while others are not projects. 
As an example, awesome microservice is not a project, but a collection of technologies that could be used to develop microservice-based systems. 

Could you please send me a complete PR? You need to fork this project, modify the HTML page accordingly and then send the PR.

sabahatatta commented:
> Thanks for your PR.
> I have added the ftgo application.
> 
> there are several interesting projects in your list that could be added, while others are not projects.
> As an example, awesome microservice is not a project, but a collection of technologies that could be used to develop microservice-based systems.
> 
> Could you please send me a complete PR? You need to fork this project, modify the HTML page accordingly and then send the PR.

Thank you for your valuable comment. I have removed these types of links and added the valid project in the list. Committing it and sending a complete PR again.

sabahatatta committed:
Update README.md
